app/src/main/java/com/example/cr554/quizapp/MainActivity.java

```diff
@@ -1,19 +1,26 @@
 package com.example.cr554.quizapp;
 
+import android.provider.MediaStore;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
 import android.view.View;
+import android.widget.RadioButton;
+import android.widget.TextView;
+
+import org.w3c.dom.Text;
 
 public class MainActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        //probably terrible performance in large enough quantities
```

Comment:
// probably ... 
  ^-- spacing

In general, most java style guides will ask for a space there ^^